Makefile

```diff
@@ -17,6 +17,9 @@ install:
 test:
```

Comment:
Ah lol, nice catch. I always just run `npm test`, so didn't run into that. Will fix.